.github/workflows/validate-license-data.yml

```
@@ -0,0 +1,33 @@
+name: Validate Licenses
+
+on:
+  pull_request:
+    branches: [ main ]
```

Changes requested: Only run validation when `licenses.json` changes

```suggestion
    branches: [ main ]
    paths:
      - 'client/src/data/licenses.json'
```